core/wallet/transaction.go

```diff
@@ -147,6 +147,10 @@ func ConfirmDepositProcess(ctx context.Context, n *core.IpfsNode, prepareRespons
 		if err != nil {
 			continue
 		}
+		err = save(n.Repo.Datastore(), confirmDepositResponse.SuccessChannelState.Channel)
+		if err != nil {
+			log.Debug(err)
```

Comment:
continue on next try?

Reply:
persisting channel_state shouldn't affect the main process even when there is an error.